market/uipath/meta.json

```diff
@@ -2,6 +2,7 @@
 	"id": "uipath",
 	"name": "UI Path RPA",
 	"listed": false,
+	"tags": ["connector"],
```

Comment:
"RPA" could be added ... since we want to put emphasize our robotic capabilities towards LTS 10